Edit requirements

Marcellii opened the pull request:
Edit the Requirements in the Documentation. Hope this is correct. If not,
please let me know.

Please success the Pull Request. Thanks.

Commit:
Edit requirements
Edit the Requirments in the Documentation. Hope this is correct. If not,
please let me know.

bastelfreak commented on thesis-de/content-de.tex:
```diff
@@ -110,6 +110,14 @@ \subsection{Datenerfassungssysteme}
 \subsection{Datenbanksysteme}
 
 \subsection{Frontends}
+Die Weboberfläche muss ausgewählten Benutzern eine Übersicht über die Nutzung einzelner Virtuellen oder Physische Maschinen bereitstellen.
```
please don't use more than 80 chars in a row, vim can highlight you these violations by setting:
`:set cc=80`

bastelfreak replied:
Virtuellen -> virtueller

bastelfreak replied:
Physische -> physischen

bastelfreak replied:
"Nutzung?" please more details.

bastelfreak commented on thesis-de/content-de.tex:
```diff
@@ -110,6 +110,14 @@ \subsection{Datenerfassungssysteme}
 
 \subsection{Frontends}
+Die Weboberfläche muss ausgewählten Benutzern eine Übersicht über die Nutzung einzelner Virtuellen oder Physische Maschinen bereitstellen.
+Hierbei ist wichtig, das die Sicherheit der Daten gewährleistet wird. Dies bedeutet, das sowohl die Verbindung zwischen Datenbank
```
"das die" -> "dass die"

bastelfreak replied:
What kind of security do we want to ensure?
remember the three As:
- Authentication
- Authentication
- Authorisation

bastelfreak commented on thesis-de/content-de.tex:
```diff
@@ -110,6 +110,14 @@ \subsection{Datenerfassungssysteme}
+Die Weboberfläche muss ausgewählten Benutzern eine Übersicht über die Nutzung einzelner Virtuellen oder Physische Maschinen bereitstellen.
+Hierbei ist wichtig, das die Sicherheit der Daten gewährleistet wird. Dies bedeutet, das sowohl die Verbindung zwischen Datenbank
+und Weboberfläche gesichert sein muss, als auch die Weboberfläche selber, gegen Zugriff von Unbekannten. Die Weboberfläche greift mittels API
+Abfragen auf die Datenbank zu und kann somit die Sicherheit gewährleisten. Die Weboberfläche soll Administratoren bei Neukonfiguration
```
why increases the API the security?

bastelfreak replied:
what do you mean by 'Neukonfiguration'?

bastelfreak commented on thesis-de/content-de.tex:
```diff
@@ -110,6 +110,14 @@ \subsection{Datenerfassungssysteme}
+Abfragen auf die Datenbank zu und kann somit die Sicherheit gewährleisten. Die Weboberfläche soll Administratoren bei Neukonfiguration
+unterstützen und zeigt hier die Auslastung der einzelnen Systemen. Der Administrator kann so gezielt die Systeme besser auslasten und 
+verteilen. Ebenfalls kann das Frontend eine Schnittstelle für komplexe Abfragen und Analysen bereitstellen. Bei Bedarf können diese auch
+Virtualisiert ausgegeben werden.
```
do you really mean 'Virtualisiert'?

bastelfreak commented on thesis-de/content-de.tex:
```diff
@@ -110,6 +110,14 @@ \subsection{Datenerfassungssysteme}
+unterstützen und zeigt hier die Auslastung der einzelnen Systemen. Der Administrator kann so gezielt die Systeme besser auslasten und 
+verteilen. Ebenfalls kann das Frontend eine Schnittstelle für komplexe Abfragen und Analysen bereitstellen. Bei Bedarf können diese auch
+Virtualisiert ausgegeben werden.
+[MR]
```
replace it with:
\mr%

bastelfreak commented:
in general, this is a useful introduction. Please add cafeterias that we can use to evaluate different UI systems and frameworks.

Commit:
Modify the edit of the requirements
I have change and modify the comments of @bastelfreak. Please let me
know, if you find anything which is incorrect.

bastelfreak commented:
still the same typos in it...

bastelfreak commented:
we close this because #42 contains the same
